topics/metagenomics/tutorials/plasmid-metagenomics-nanopore/tools.yaml

```diff
@@ -0,0 +1,31 @@
+---
```

Comment:
@shiltemann This file should be updated, right?

Reply:
Actually we do not need this file anymore as we use the information directly from the workflow.